docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -314,6 +314,17 @@ Each tablet replica generally requires 700 MiB of this memory.
 
 Default: `0` unless [`--use_memory_defaults_optimized_for_ysql`](#use-memory-defaults-optimized-for-ysql) is true.
 
+##### tablet_replicas_per_core_limit
+
+The number of tablet replicas that each core on a YB-TServer can support. This is used by both YB-Masters and YB-TServers.
```

Comment:
I hadn't realized we have a page for yb-master flags and a page for yb-tserver flags. In that case, we should probably remove the last sentence:

> This is used by both YB-Masters and YB-TServers.

Here and elsewhere.